api-docs/rst/dev-guide/concepts.rst

```diff
@@ -798,3 +808,36 @@ To get started with RackConnect v3 cloud bursting:
 
 .. _RackConnect product information: http://www.rackspace.com/cloud/hybrid/rackconnect
 .. _Knowledge Center articles for RackConnect: http://www.rackspace.com/knowledge_center/product-page/rackconnect
+
+
+.. _concergence-concept:
+
+Convergence
+~~~~~~~~~~~
+
+Convergence is completely rewritten orchestration layer of autoscale that provides
+higher reliability for scaling up and down by optimizing use of Nova API with retries
+as necessary. In addition, Convergence continuously keeps track of all the servers
+in autoscaling groups, notices when a server is deleted out-of-band or goes into
+an non-ACTIVE state (eg ERROR) and automatically replaces it.
+We call this capability “self-healing”.
+
```

Comment:
Convergence also provides a *self-healing* capability by tracking all the servers in autoscaling groups 
continuously and automatically replaces any servers that have been deleted out-of-band or transitioned to an 
error state.